### PR TITLE
Implementation: Footer

### PR DESCRIPTION
**Pull request by divyanshkul**
### Submission Checklist

- [x] Your pull request targets the `master` branch of the repository.
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] You have read the [Contributing guidelines](https://github.com/jboss-outreach/gci/blob/master/CONTRIBUTING.md) and your changes follow them.

#### Type of Change

- [ ] Bug fix
- [x] New implementation

### PR Description

Addresses Issue https://github.com/jboss-outreach/gci/issues/76

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,9 +248,73 @@ <h2>Latest Issues</h2>
...
+          </div>
+        </div>
+        <div class="col-12 col-md-3" >
+          <span class="pull-right">
+            <div class="heading">
+              <p class="text-center">Sitemap</p><hr>
+            </div>        
+            <div class="contents text-left">
+              <a href="#welcome" class="quick-links">Home</a><br>
+              <a href="#mentors" class="quick-links">Our Mentors</a><br>
```
Comment (aashutoshrathi) [inline index.html]: Make it Mentors

File: index.html [hunk 2]
```diff
@@ -248,9 +248,73 @@ <h2>Latest Issues</h2>
...
+              <a href="#contributors" class="quick-links">Contributors</a><br>
+              <a href="#students" class="quick-links">Our brilliant students</a><br>
```
Comment (aashutoshrathi) [inline index.html]: make it `Students`

File: index.html [hunk 3]
```diff
@@ -248,9 +248,73 @@ <h2>Latest Issues</h2>
...
+              <a href="#projects" class="quick-links" >Our Projects</a><br>
```
Comment (aashutoshrathi) [inline index.html]: Make it `Projects`

File: index.html [hunk 4]
```diff
@@ -248,9 +248,73 @@ <h2>Latest Issues</h2>
...
+        <div class="social-icons align">
+          <a href="https://www.facebook.com/jboss"> <i class="fa fa-facebook-square " style="font-size: 50px;"></i></a>
+          <a href="https://twitter.com/JBoss" target="_blank"> <i class="fa fa-twitter-square " style="font-size: 50px;"></i></a>
+          <a href="https://github.com/jboss-outreach" target="_blank"> <i class="fa fa-github-square " style="font-size: 50px;"></i></a>
+          <a href="https://planet.jboss.org/" target="_blank"> <i class="fa fa-rss-square" style="font-size: 50px"></i></a>                  
+        </div>
+      </div>
+      <div class="row" style="margin-right: -190px;"">
+        <div class="text-center copyright" style=" margin-left: 30px;">
+        <br>Copyright &copy; <a href="https://github.com/jboss-outreach" class="community-link">Jboss Communty</a> - 2018 </div>
```
Comment (aashutoshrathi) [inline index.html]: Fix the typo, it should be `JBoss Community`.

**Comment by divyanshkul** [comment]
Okay, on it @aashutoshrathi

File: index.html (hunk 4 above)
Comment (divyanshkul) [reply index.html]: Sorry for this, silly mistake 😏

**Comment by divyanshkul** [comment]
Done @aashutoshrathi do have a look now. :)

**Comment by aashutoshrathi** [comment]
Shift `Our Google Code-In 2017 Journey` to left by decreasing container size of About Us

**Comment by divyanshkul** [comment]
Hey @aashutoshrathi does this [latest build](https://bot.jboss-outreach.org/2018-01-05/978c57d) seems fine or should I shift it more further? I decreased the size of the about us column and added equal space between each of them for uniformity. 
Good Day :)

File: index.html [hunk 5]
```diff
@@ -248,9 +248,73 @@ <h2>Latest Issues</h2>
...
+        <div class="social-icons align">
+          <a href="https://www.facebook.com/jboss"> <i class="fa fa-facebook-square " style="font-size: 50px;"></i></a>
+          <a href="https://twitter.com/JBoss" target="_blank"> <i class="fa fa-twitter-square " style="font-size: 50px;"></i></a>
+          <a href="https://github.com/jboss-outreach" target="_blank"> <i class="fa fa-github-square " style="font-size: 50px;"></i></a>
+          <a href="https://planet.jboss.org/" target="_blank"> <i class="fa fa-rss-square" style="font-size: 50px"></i></a>
+        </div>
+      </div>
+      <div class="row">
+        <div class="text-center copyright" style=" margin-left: 30px;">
+        <br>Copyright &copy; <a href="https://github.com/jboss-outreach" class="community-link">Jboss Community</a> - 2018 </div>
```
Comment (aashutoshrathi) [inline index.html]: Please make it `JBoss` from `Jboss`

**Comment by divyanshkul** [comment]
@aashutoshrathi 
* Reduced the size of all the contents by **2px.**
* Centered the heading ```contact us``` :)

**Comment by aashutoshrathi** [comment]
Nice work, sorry I wasn't available yesterday

**Comment by divyanshkul** [comment]
No worries @aashutoshrathi we all have our lives outside GCI as well. ;)

**Comment by divyanshkul** [comment]
No worries @aashutoshrathi we all have our lives outside GCI as well. 😉

**Pull request merged**